spec/services/dwp/benefit_check_service_spec.rb

```diff
@@ -1,35 +1,131 @@
 require 'rails_helper'
+require 'savon/mock/spec_helper'
 
 RSpec.describe DWP::BenefitCheckService do
-  subject { described_class.call(applicant) }
+  subject { described_class }
 
-  let(:last_name) { 'Smith' }
-  let(:date_of_birth) { '1999/01/11'.to_date }
-  let(:nino) { 'NC123459A' }
+  let(:last_name) { 'Walker' }
+  let(:date_of_birth) { '1980/01/10'.to_date }
+  let(:nino) { 'JA293483A' }
+  let(:id) { '123' }
 
   let(:applicant) do
     double(
       Applicant,
+      id:,
       last_name:,
       date_of_birth:,
-      nino:
+      nino:,
     )
   end
 
+  before do
+    Rails.configuration.x.benefit_checker.use_mock = false
+    Rails.configuration.x.benefit_checker.wsdl_url = 'http://benefit-checker/?wsdl'
+  end
+
   describe '.call' do
+    before do
+      stub_request(:post, 'http://benefit-checker/?wsdl')
+        .to_return(body: benefit_checker_fixture)
+    end
+
+    let(:expected_params) do
+      hash_including(
+        message: hash_including(
+          clientReference: applicant.id,
+          surname: applicant.last_name.strip.upcase,
+          dateOfBirth: applicant.date_of_birth.strftime('%Y%m%d'),
+        ),
+      )
+    end
+
+    let(:benefit_checker_fixture) { File.read('spec/fixtures/benefit_checker_responses/successful.xml') }
```

Comment:
If we place the fixtures inside the existing `spec/fixtures/files` then we can use the helper `file_fixture('benefit_checker_responses/successful.xml').read`
And we keep it tidy and organised. There are already fixtures for address lookups.